gapic/templates/%namespace/%name_%version/%sub/services/%service/client.py.j2

```diff
@@ -402,9 +466,19 @@ class {{ service.client_name }}(metaclass={{ service.client_name }}Meta):
             self._client_options = client_options_lib.ClientOptions()
         self._client_options = cast(client_options_lib.ClientOptions, self._client_options)
 
-        self._use_client_cert, self._use_mtls_endpoint = {{ service.client_name }}._read_environment_variables()
+        try:
+            universe_domain_opt = getattr(self._client_options, 'universe_domain')
```

Comment:
In the `__init__` docstring above, the section for `client_options` says " It won't take effect if a ``transport`` instance is provided." That should be updated, and the description should have a third clause explaining the new ``universe_domain`` property.

Reply:
addressed.

Reply:
Replicated docstring for async client (and updated other parts of the docstring that were out of date).